library/src/main/java/com/datatorrent/lib/io/block/BlockMetadata.java

```diff
@@ -227,37 +226,10 @@ public String getFilePath()
       return filePath;
     }
 
-    /**
-     * Get if blocks should be read in sequence
-     * @return readBlockInSequence
-     */
-    public boolean isReadBlockInSequence()
```

Comment:
I mentioned in the discussion that the way this was accomplished is hacky.
The correct way of doing this is by setting a StreamCodec which I changed in FSInputModule